cirrus/server/tests/test_experiment_recipes.py

```diff
@@ -4,11 +4,16 @@
 import requests
 
 from cirrus.experiment_recipes import RemoteSettings
+from cirrus.sdk import SDK
+from cirrus.settings import context
 
 
 class TestFetchRecipes(unittest.TestCase):
+    def setUp(self):
+        self.sdk = SDK(context)
```

Comment:
a wild Camel appears 
```suggestion
    def set_up(self):
        self.sdk = SDK(context)
```

Reply:
Sadly, the builtin methods in the Python unittest framework use [camelcase](https://docs.python.org/3/library/unittest.html#unittest.TestCase.setUp).

Reply:
I completely understand @jrbenny35 how weird it looks in python code, but builtin methods :(